libs/utils/lana.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
Is this needed for the whole file? I'd rather have ESLint disabled for specific purposes, otherwise we might miss a lot of errors.